media/js/firefox/new/scene1_fxa_modal.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This relies on base/mozilla-modal.js, included in the firefox_new_scene1 bundle
+
+(function($) {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+    var $downloadButton = $('#download-button-desktop-release .download-link');
```

Comment:
Will clicking this link register as a download event in GA even though it may open a modal? Do we need to do anything special here to avoid confounding the data?

Reply:
I have no idea.

Reply:
I asked this same question in the pre-download PR, and Justin provided some insight:

"Only one download per session counts as a download in our reporting. We often see sessions clicking 2, 3, 6, 8 times. Those count as one download."

So, I think we're good here 👍